train.py

```diff
@@ -625,10 +661,11 @@ def run(cfg: Any) -> None:
                     param.requires_grad = True
                     param.data = param.data.float()
 
-    if cfg.environment._distributed:
+    if cfg.environment._distributed and not cfg.environment.use_deepspeed:
         model = wrap_model_distributed(model, cfg, cfg.environment.use_fsdp)
 
-    if cfg.environment.compile_model:
+    # deepspeed do not support torch.compile
+    if cfg.environment.compile_model and not cfg.environment.use_deepspeed:
```

Comment:
throw a warning, if the compile was selected with deepspeed

Reply:
fixed

[67629ee](https://github.com/h2oai/h2o-llmstudio/pull/288/commits/67629eee9dd873992b035626d0fe2ee9f23f7080)